scene/2d/camera_2d.cpp

```diff
@@ -650,15 +650,25 @@ void Camera2D::align() {
 	_update_scroll();
 }
 
+// deprecated
 void Camera2D::set_position_smoothing_speed(real_t p_speed) {
 	position_smoothing_speed = MAX(0, p_speed);
 	_update_process_internal_for_smoothing();
 }
 
+// deprecated
```

Comment:
There actually is a standardized way to deprecate things. Look up how Control's `auto_translate` is deprecated, or for an example of my own https://github.com/godotengine/godot/pull/93940 .

The idea is that, compiled with the right flags, the engine can run without the deprecated things even existing in the code.

Reply:
Ok, thank you i will check that out.